content/en/docs/instrumentation/java/getting-started.md

```diff
@@ -146,19 +224,24 @@ For more:
 - Learn about [manual instrumentation][] and try out more
   [examples]({{% relref examples %}}).
 
+## Troubleshooting
```

Comment:
There already is a troubleshooting section here: https://opentelemetry.io/docs/instrumentation/java/automatic/#troubleshooting

Reply:
I put it here to be consistent with the JavaScript documentation, but let me remove it for now, we don't need to have it twice